pkg/topology/query.go

```diff
@@ -1,231 +1,65 @@
 package topology
 
 import (
-	"context"
 	"fmt"
 	"net/url"
 	"strconv"
 	"strings"
 
-	jsontime "github.com/liamylian/jsontime/v2/v2"
-	"github.com/pkg/errors"
-
-	"github.com/flanksource/canary-checker/pkg"
 	"github.com/flanksource/canary-checker/pkg/db"
-	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/commons/collections"
 	"github.com/flanksource/duty"
+	"github.com/flanksource/duty/models"
 )
 
-var json = jsontime.ConfigWithCustomTimeFormat
-
-const DefaultDepth = 1
-
-func parseItems(items string) []string {
-	if strings.TrimSpace(items) == "" {
-		return nil
-	}
-	return strings.Split(items, ",")
-}
+const DefaultDepth = 3
 
 type TopologyParams struct {
```

Comment:
Can we move this to duty as well as we would want to switch the api call to incident-commander so that we can still return the topology even if canary checker is down

Reply:
Have removed this struct, we now use duty.TopologyParams everywhere